rewards/aws/boost.py

```diff
@@ -15,9 +15,11 @@ def upload_boosts(boost_data, chain: str):
     """
     chain_id = env_config.get_web3(chain).eth.chain_id
     boost_file_name = f"badger-boosts-{chain_id}.json"
-    buckets = ["badger-staging-merkle-proofs"]
+    buckets = []
     if not env_config.test:
```

Comment:
I’d prefer if this was reversed (checking if not something is bad for readability) and a production specific flag has to be set. Currently there’s a fall through just checking test, if something goes wonky there then it defaults to uploading to prod. 

So something like:
```
if env_config.production:
    buckets.append(prod bucket)
elif env_config.staging:
    buckets.append(staging bucket)
elif env_config.test
    buckets.append(test bucket)
else
    raise config error
```